apps/trading-e2e/src/integration/deposits.feature

```diff
@@ -0,0 +1,113 @@
+Feature: Deposits to vega wallet
+
+  Background:
+    Given I navigate to deposits page
+
+  @todo
+  Scenario: Eth wallet connect message displayed if no ethereum wallet connected
+    Then I can see a message stating no wallet is connected
+    And the connect button is displayed
+
+  @todo
+  Scenario: Cannot deposit if approved amount is 0 (approval amount is 0)
+    And I connect my ethereum wallet
```

Comment:
Do we need a line for `Given I am on the deposit page` at the start of each of these scenarios?